docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+illustrating to access the EESSI software stack via a small container provided by
+EESSI. The container only contains a minimal number tools such as the CernVM-FS
+client and some configuration data.
+
+### Prerequisites
+
+- Singularity 3.7.x or newer _or_ Apptainer x.y.z or newer (any version with
+  support for `--fusemount` should be sufficient)
+- `git` available on your system
+
+### Preparation
+
+Clone the [`EESSI/software-layer`](https://github.com/EESSI/software-layer.git)
+repository to a directory `eessi_container` by running the command
+```bash
+git clone https://github.com/EESSI/software-layer.git eessi_container
+```
+Change directory
+```bash
+cd eessi_container
+```
+
+## Basic scenarios
+
+### Get access to EESSI
+
+Simply run the command (from the directory you have changed into above)
+```bash
+./eessi_container.sh
+```
+_(Note, the first run may take a bit longer because the container image is
+downloaded and converted.)_
+
+You should see output like
+```
+Using /tmp/eessi.RC6kYEjmSj as tmp storage (add '--resume /tmp/eessi.RC6kYEjmSj' to resume where this session ended).
+Launching container with command (next line):
+singularity shell --fusemount container:cvmfs2 pilot.eessi-hpc.org /cvmfs/pilot.eessi-hpc.org docker://ghcr.io/eessi/build-node:debian11
+CernVM-FS: pre-mounted on file descriptor 3
+Singularity> CernVM-FS: loading Fuse module... done
+fuse: failed to clone device fd: Inappropriate ioctl for device
+fuse: trying to continue without -o clone_fd.
+
+Singularity>
+```
+_(Note, you may have to press enter to clearly see the prompt as some messages
+beginning with `CernVM-FS: loading Fuse` have been printed after the first prompt
+`Singularity> ` was shown.)_
+To access the EESSI pilot stack just run the command
+```
+source /cvmfs/pilot.eessi-hpc.org/latest/init/bash
+```
+This may take a while as data is downloaded from Stratum 1 server which is part of the
+CernVM-FS infrastructure to distribute files. You should see the following (or
+similar output depending on the CPU architecture of the machine you are accessing
+EESSI).
+```
+Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/versions/2021.12!
+archspec says x86_64/intel/skylake_avx512
+Using x86_64/intel/skylake_avx512 as software subdirectory.
+Using /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/modules/all as the directory to be added to MODULEPATH.
+Found Lmod configuration file at /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/.lmod/lmodrc.lua
+Initializing Lmod...
+Prepending /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/modules/all to $MODULEPATH...
+Environment set up to use EESSI pilot software stack, have fun!
+[EESSI pilot 2021.12] $ 
+```
+The last line is the prompt. Run `module avail` to see which modules and
+extensions are available. About 280 modules and 1800 extensions are installed.
```

Comment:
I wouldn't mention numbers here, nightmare to keep up-to-date...

Reply:
Removed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001